src/hooks.js

```diff
@@ -16,22 +17,22 @@ const toJson = (item) => {
 }
 
 /**
- * Move 'jwt-auth' key to aio-cli-config
+ * Move all keys to aio-cli-config
  *
  * @param {Function} debug
  */
 const upgrade = () => {
   try {
     const oldConf = new OldConf({ projectName: '@adobe/aio-cli-plugin-config' })
+    let data = oldConf.store
+    delete data['__backup__']
 
-    let data = oldConf.get('jwt-auth')
-    oldConf.delete('jwt-auth')
-
-    if (data == null) return
-
-    oldConf.set('jwt-auth-backup', data)
-    config.set('jwt-auth', toJson(data))
-    debug('config upgraded')
+    if (data != null && Object.keys(data).length > 0) {
```

Comment:
the delete above would throw if data was null. I would move the delete into the if statement